app/reviews/models.py

```diff
@@ -26,6 +26,9 @@ class Review(TimestampMixin):
 
     # Make nullable for anonymity of reviewers.
     user = models.ForeignKey(User, on_delete=models.SET_NULL, blank=True, null=True)
+    # anonymity_id is just a unique identifier for the reviewer used
+    # as a check prevent multiple reviews from one person
+    anonymity_id = models.CharField(blank=True, null=True, max_length=250)
```

Comment:
Based off of our discussion, I agree we should be flexible and just leave the responsibility of one user one review to the clients.